data-prepper-plugins/drop-events-processor/src/test/java/org/opensearch/dataprepper/plugins/processor/drop/DropEventProcessorConfigTest.java

```diff
@@ -29,7 +28,7 @@ void getWhen() {
 
     @Test
     void getHandleFailedEventsOption() {
-        assertThat(config.getHandleFailedEventsOption(), is(HandleFailedEventsOption.SKIP));
+        assertThat(config.getHandleFailedEventsOption(), CoreMatchers.is(HandleFailedEventsOption.SKIP));
```

Comment:
Why the change here? There's a static import for `CoreMatchers.is` already.

Reply:
I'm not quite sure. I used IntelliJ's "Optimize Imports" here. I fixed it in the latest commit.